src/graphql/schema.ts

```diff
@@ -61,7 +64,7 @@ export default class Schema {
   }
 
   public getType(name: string, allowNull: boolean = false): GraphQLType | null {
-    name = upcaseFirstLetter(name);
+    name = this.prepareTypeFunc(name);
```

Comment:
I think we can call `Context.getInstance().adapter.prepareSchemaTypeName` here and omit the `prepareTypeFunc` param.

Reply:
Fixed :)